manifests/config.pp

```diff
@@ -35,6 +35,7 @@
       owner   => $::vault::user,
       group   => $::vault::group,
       mode    => $::vault::config_mode,
+      notify  => Class['vault::service'],
```

Comment:
Can you please split this out into its own commit as it addresses a different issue.

Reply:
I don't really understand why it should be a different commit. As when you update the binary you need to restart the service. I can do it, just want to understand why :)